build_tools/code_sign_osx.py

```diff
@@ -15,13 +15,44 @@
     "SasView*.app/Contents/Resources/zmq/.dylibs/*.dylib", recursive=True
 )
 
+pyside_QtWebEngineProcessApp = glob.glob(
+    "SasView*.app/Contents/Resources/PySide6/Qt/lib/QtWebEngineCore.framework/Versions/A/Helpers/QtWebEngineProcess.app", recursive=True
+)
+
+pyside_QtWebEngineCore = glob.glob(
+    "SasView*.app/Contents/Resources/PySide6/Qt/lib/QtWebEngineCore.framework/Versions/A/QtWebEngineCore", recursive=True
+)
+
+pyside_QtWebEngineProcess_Helpers = glob.glob(
+    "SasView*.app/Contents/Resources/PySide6/Qt/lib/QtWebEngineCore.framework/Helpers/QtWebEngineProcess.app/Contents/MacOS/QtWebEngineProcess", recursive=True
+)
+
+pyside_Qtlibs = glob.glob(
+    "SasView*.app/Contents/Resources/PySide6/Qt/lib/Qt*.framework/Versions/A/Qt*", recursive=True
+)
+
+#pyside_libs = pyside_QtWebEngineCore + pyside_QtWebEngineProcess
```

Comment:
Why do we need to notarize Qt libraries? They are already signed by the Qt Company, after all?
Unless this is something very macos specific? But then, we don't notarize other libraries we depend on...

Reply:
@rozyczko: Yeah, good question. Once they are touched e.g. packaged with pyinstaller they need to be signed again.